### PR TITLE
Spanish, german and some fixes to classes and template

### PR DESCRIPTION
**Pull request by JJBocanegra**
German translation thanks to @CarnVanBeck

### CODE REVIEW

File: lang/en.json [hunk 1]
```diff
@@ -57,6 +57,8 @@
...
     "CMPBrowser.spellBrowser":"Spell Browser",
     "CMPBrowser.npcBrowser":"NPC Browser",
+    "CMPBrowser.featBrowser": "Feat Browser",
+    "CMPBrowser.itemBrowser": "Item Browser",
     "CMPBrowser.settings":"Settings"
-    
-}
\ No newline at end of file
+
+}
```
Comment (spetzel2020) [inline lang/en.json:64]: You've added references to CMPBrowser.rogue, .fighter, .monk without defining them in any of the provided language files.

**Review by spetzel2020 — CHANGES_REQUESTED** [review]
Other than the references to CMPBrowser.fighter etc, I only formatting changes; am I missing anything.

**Comment by spetzel2020** [comment]
Alternatively I can make the fixes and do a manual merge; let me know.

**Comment by spetzel2020** [comment]
I've gone through this in more detail and think it will be a lot of work for you to re-pull this against the current version. Instead I will cherry-pick the PR and credit you appropriately in the release v0.6. Thanks!

**Comment by spetzel2020** [comment]
Cherry-picked into commit 9d72f1bdac620ca8d03da42e70238653dedd6b5b

**Comment by JJBocanegra** [comment]
Sorry I just noticed these messages @spetzel2020! Thanks for your hard work!

**Pull request closed without merging**